src/main/java/exnihilocreatio/registries/registries/FluidBlockTransformerRegistry.java

```diff
@@ -47,16 +54,17 @@ public boolean canBlockBeTransformedWithThisFluid(Fluid fluid, ItemStack stack)
         return false;
     }
 
-    public ItemInfo getBlockForTransformation(Fluid fluid, ItemStack stack) {
-        ItemInfo info = ItemInfo.getItemInfoFromStack(stack);
+    @Nonnull
+    public BlockInfo getBlockForTransformation(Fluid fluid, ItemStack stack) {
```

Comment:
Isn't this breaking compat? but not sure whether any mod uses it anyways

Reply:
I've checked these changes with ExCompressum, which is the main and only mod I know that uses ExNihilo and there's no error or crashes of note.

Reply:
there is also TinkersComplement, but that's all I know about

Reply:
I can add that to my dev space and test that too if needed